### ISSUE TITLE
After one idle day, a 2 USDT invest swallows the full TTS drip

### ISSUE THREAD

**Issue by sancunshiguang** [ttswap/ttswap-core#292]
**Root cause.** `Token._stakeFee` on `stake`/`unstake` writes `leftamount/18250` (or the 1e12 floor) into `poolstate.amount0` when `lasttime + 86400 < now`, **without requiring any stake to exist**. `investGood` on a value/promised good calls `Token.stake`. The first (or ~100% share) staker immediately `disinvestProof` → `unstake`, and `profit = pool0 * proof / stake1` mints **the entire pool drip** as TTS.

Cross-contract: `Market.investGood` → `Token.stake` → `_stakeFee` credits the pool → `Market.disinvestProof` → `Token.unstake` mints.

**Attack path.**
1. Wait until `stakestate.amount0() + 86400 < block.timestamp` (one day after deploy, or one day after everyone has unstaked).
2. `investGood(USDT, qty≈2e6)` (value good, value ≥ `MIN_INVEST_VALUE`).
3. Same tx or next: `disinvestProof(all shares)`.
4. Recover USDT (minus ~8+8 bps); pocket the full day's TTS mint.

**Preconditions.** USDT (or any good) is marked value good / promised. No role. Smaller `stake1` is better; at 0 the attacker takes 100%.

**Calldata.**

```
// t0 + 86401
investGood(
  T_GoodKey(1, USDT, 0),
  toTTSwapUINT256(0, 2e6),
  0x, 0x, attacker
)
disinvestProof(proofId(attacker, usdtGoodId), shares, address(0), attacker, 0x)
```

**PoC** — `test_RT15_idle_drip_captured_by_dust_value_invest` PASS:

```
RT15 TTS minted (raw 12d): 10958904109589041   // ≈ 10,958.9 TTS
RT15 USDT spent (raw 6d):  3199                 // 0.003199 USDT
```

**Attacker profit.** Capital 0.003 USDT, output 10,958 TTS (`2e20/18250`, ratio=10000). Repeatable daily while the attacker remains the largest/only staker. Protocol loss = unbacked mint.

**Fix sketch.** `_stakeFee` must not credit `poolstate` when `stakestate.amount1()==0`; or pay the drip from a pre-minted reserve; or decouple value-good invest from stake.
